File: tmp-mei/fgen4255027576711118260/base/test/ResultPrinter_0.java
package base.test;
public interface ResultPrinter_0 extends base.Sealed_0,base.test.ResultReporter_0{
ResultPrinter_0 $self = new ResultPrinter_0Impl();
base.Void_0 $hash$mut(base.List_1 results_m$);
static base.Void_0 $hash$mut$fun(base.List_1 results_m$, base.test.ResultPrinter_0 $this) {
  return ((base.Void_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Void_0) null;
}})
);
}
}

File: tmp-mei/fgenjs5030319945734616766/base/test/ResultPrinter_0.js
import { base$$test$$ResultReporter_0 } from "../../base/test/ResultReporter_0.js";
import * as rt from "../rt/main.js";

export class base$$test$$ResultPrinter_0 extends base$$test$$ResultReporter_0 {
  static $self = new base$$test$$ResultPrinter_0();

  $hash$mut(results_m$) {
  return base$$test$$ResultPrinter_0.$hash$mut$fun(results_m$, this);
}

  static $hash$mut$fun(results_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}
